ReadMe.md

```diff
@@ -0,0 +1,215 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Math Magician App <a name="about-project"></a>
+
+> "Bookstore" is a website for book lovers. It is a Single Page App (SPA) that allows users to:
+
+- Make simple book addtion to their lists.
+- You can edit, remove and add.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Javascript
+- React.js
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a href="https://reactjs.org/docs/create-a-new-react-app.html">React</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **This is a SPA.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://the-normal-one1.github.io/bookstore/)
```

Comment:
- As per the previous reviewer [here](https://github.com/amklass11/Bstore/pull/4/files#r1125419066) kindly remove this link completely as you currently don't have a live link